zulipterminal/config/markdown.py

```diff
@@ -0,0 +1,86 @@
+from typing import List
+
+from typing_extensions import TypedDict
+
+
+class MarkdownElements(TypedDict, total=True):
+    help_text: str
+    raw_text: str
+    html_element: str
+
+
+MARKDOWN_ELEMENTS: List[MarkdownElements] = [{
+        'help_text': 'Title text',
+        'raw_text': 'You type',
+        'html_element': 'You get',
```

Comment:
I see how this makes it easier to add the column titles, but it's confusing.

I'd suggest manually adding these column headings; really it would be better if these column headings stayed in place as things scrolled, though that would likely need adjustment in the main popup code.